### PR TITLE
Kill prop-types

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

**Comment by saghul** [comment]
Sigh. Unfortunately the latest version has a very nasty animation we don't want to get, but at the same time any older version I try give me an error in i18n so flags don't work at all. I guess we are stuck with this for the time being. 😞

**Comment by virtuacoplenny** [comment]
tldr Maybe we can disable the slide in animation? Maybe the slide in animation doesn't buy enough if it's preventing the update, especially considering longer term goals for the UI packages being used.

```
_atlaskit_overrides.scss

.gXSEsl:first-child {
    animation: none !important;
    -webkit-animation: none !important;
}
```

Ramblings:

This is what I've learned so far. This issue start with @atlaskit/flags version 6.1.6, which updated react-transition-group, which is used for changing classnames on the Flag after the slide in animation ends. 

Flag animation `cbfRuT` seems to be part of the issue. `cbfRuT` is an animation that positions a Flag offset to the left so it can be seen moving right-ward into place. That animation is used in the class `.gXSEsl:first-child`; `.gXSEsl` is the class put onto each Flag that has not completed its slide in animation yet while `:first-child` is the pseudo selector for the first Flag in the Flags Container. So whenever a Flag with `.gXSEsl` becomes the first Flag in a Flags Container, animation `cbfRuT` will play.

Normally when the slide-in animation ends, I think the Flag class changes from `.gxSEsl` to `.byiuGy`, preventing `.gXSEsl:first-child` from selecting the Flag because it doesn't have the `.gXSEsl` class. This can work for the atlassian use case (for the most part, assuming animations are allowed to end) because Flags are put in a stack of first in, last out. So when Example Flag A displays, it plays animation `cbfRuT` and goes from `.gXSEsl` to `.byiuGy`. When Example Flag B displays and is dismissed, Example Flag A is already set to `.byiuGy` so it doesn't animate again when it becomes the first Flag again.

This does not work for the jitsi use case because Flags are put in a queue. Example Flag A will display and go from `.gXSEsl` to `.byiuGy`. Example Flag B will be queued and left with class  `.gXSEsl`. So when Example Flag A is dismissed, Example Flag B becomes the first child and animates and then changes to `.byiuGy`.

Disclaimer: I could be absolutely talking nonsense but it's making sense to me at this very moment at least.
